library/std/src/fs.rs

```diff
@@ -872,6 +901,9 @@ impl Seek for File {
     fn seek(&mut self, pos: SeekFrom) -> io::Result<u64> {
         (&*self).seek(pos)
     }
+    fn stream_len(&mut self) -> io::Result<u64> {
+        (&*self).stream_len()
```

Comment:
You've missed the `Seek` implementation for `Arc<File>` below

Reply:
Thanks for noticing. Added.